dev_guide/projects.adoc

```diff
@@ -194,3 +194,160 @@ To delete a project using the CLI:
 ----
 $ oc delete project <project_name>
 ----
+
+ifdef::openshift-online[]
+[[project-collaboration-in-online-pro]]
+== Project Collaboration in {product-title} Pro
+
+Every {product-title} Pro account has the ability to add up to 50 collaborator
+users per subscription. These collaborator users are granted cluster access from
+{product-title} Pro account subscribers so that they can collaborate on projects
+hosted with {product-title}. This allows multiple users to gain access to
+projects under a single subscription, without having to pay a monthly fee for
+every account.
+
+[NOTE]
+====
+Existing {product-title} Pro users do not need to be added as collaborators.
+====
+
+Collaboration is not available in {product-title} Starter.
+
+[[collaboration-restrictions]]
+=== Collaboration Restrictions
+
+Collaborators can only access the resources within the projects that they have
+been granted access to. Also, their ability to view, edit, and manage the
+project resources will depend on the specific role that they have been granted
+within the project.
+
+[[project-collaboration-adding-collaborators]]
+=== Adding Collaborators
+{product-title} Pro subscribers can add collaborators by following these steps:
+
+. Each user you want to add as a collaborator must create a free account at
+ link:https://developers.redhat.com/[developers.redhat.com]. Once your
+ collaborator has confirmed their Red Hat Developers account, you can add them
+ to your subscription.
+
+ . Each collaborator must sign into
+link:https://developers.redhat.com/[developers.redhat.com] and click on their
+name in the upper-right corner to access their account details. Make note of the
+*Red Hat Login ID* on this page, as it is the user name you will be required to
+enter in order to associate the collaborator with your subscription.
+
+. Sign in to link:https://manage.openshift.com[manage.openshift.com] and click on
+*Manage Subscription* under the cluster you want to add the collaborator to.
++
+image::manage-subscriptions-collaboration.png[]
+
+. Once you are in the subscription management console, click the *Manage* link
+under the *Collaborators* heading, which brings you to the *Collaboration* page.
++
+image::manage-collaborators.png[]
+
+. On the *Collaboration* page, enter the *Red Hat Login ID* for the user in the user
+name field and click *Add Collaborator*.
++
+You should now see the user listed under your collaborators, the time the user
+was added, and an option to remove the user from your subscription.
++
+This does not automatically grant the user any access to your projects. Access
+must be granted manually by the project owner using
+xref:granting-project-access-using-the-cli[`oc policy` commands] or the
+xref:granting-project-access-using-the-web-console[web console].
```

Comment:
Should we point to https://access.redhat.com/solutions/3093811 as docs for how to do this?

Reply:
Or is this KCS now deprecated by the next section[s] of docs?

Reply:
@sferich888 Doesn't CEE already have tier-2 access to be able to access all resources within a user's projects on all Pro clusters?

Reply:
@sferich888 I think this PR deprecates that KCS